packages/flutter_tools/lib/src/commands/analyze_once.dart

```diff
@@ -16,10 +16,8 @@ import '../base/platform.dart';
 import '../base/terminal.dart';
 import '../base/utils.dart';
 import '../dart/analysis.dart';
-import 'analyze.dart';
 import 'analyze_base.dart';
 
-/// An aspect of the [AnalyzeCommand] to perform once time analysis.
```

Comment:
Why remove the doc comment?

Reply:
AnalyzeOnce class name is enough, No additional comments.

Reply:
Yeah you're right, its not a very helpful comment